flow/connectors/postgres/postgres.go

```diff
@@ -620,6 +621,11 @@ func (c *PostgresConnector) getTableSchemaForTable(
 
 		columnNames = append(columnNames, fieldDescription.Name)
 		columnTypes = append(columnTypes, string(genericColType))
```

Comment:
Remove `ColumnNames`/`ColumnTypes`. Maybe as a separate PR preceding this one where you only introduce TypeModifier but don't really use it

Reply:
Removing those two involves a wide refactor of e2e and merge statements across peers. Would prefer doing it as a follow-up PR

Reply:
It should be done as a preceding PR